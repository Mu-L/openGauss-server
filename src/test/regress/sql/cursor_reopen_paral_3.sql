start transaction;
select pg_sleep(6);
call proc1();
select * from employees;
commit;