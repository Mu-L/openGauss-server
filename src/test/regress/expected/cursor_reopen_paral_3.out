start transaction;
select pg_sleep(6);
 pg_sleep 
----------
 
(1 row)

call proc1();
INFO:  v_lastname is wang
INFO:  v_jobid is 1
INFO:  ----------------------------
INFO:  v_age is 1
INFO:  v_depid is 1
INFO:  ----------------------------
 proc1 
-------
 
(1 row)

select * from employees;
 last_name | job_id 
-----------+--------
 wang      |      1
 hu        |      2
 zhou      |      3
 wang      |     11
 wang      |    101
 wang      |     11
 wang      |     11
(7 rows)

commit;
